api-server/src/server/index.js

```diff
@@ -89,6 +82,31 @@ app.start = _.once(function () {
   });
 });
 
+if (sentry.dsn === 'dsn_from_sentry_dashboard') {
+  log('Sentry reporting disabled unless DSN is provided.');
+} else {
+  Sentry.init({
+    dsn: sentry.dsn,
+    integrations: [
+      new Sentry.Integrations.Http({ tracing: true }),
+      new Tracing.Integrations.Express({
+        app
+      })
+    ],
+    // Capture 20% of transactions to avoid
+    // overwhelming Sentry and remain within
+    // the usage quota
+    tracesSampleRate: 0.2
+  });
+  log('Sentry initialized');
+}
+
+// RequestHandler creates a separate execution context using domains, so that every
+// transaction/span/breadcrumb is attached to its own Hub instance
+app.use(Sentry.Handlers.requestHandler());
+// TracingHandler creates a trace for every incoming request
+app.use(Sentry.Handlers.tracingHandler());
```

Comment:
I think we should be consistent with the middleware. I'd put it all in api-server/src/server/middleware.json and let loopback handle 'use'ing it.

I'm not sure what the consequences of 'use'ing the request handler twice are, but I'd rather we didn't.

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/api-server/src/server/middlewares/sentry-request-handler.js

Reply:
That's a good point. We could make a proper "loopback" middleware approach here, but I think if we are to move to new-API, this will be carried over. 

IDK I am comfortable moving it to the middleware-style implementation if you want?

Reply:
Since loopback is being a pain right now, I'd be more comfortable if we did things in the way it expects (i.e. in middleware.json). I'm happy to push a commit with that, if you like.

I agree that it's more work to carry over to the new api, but it's not much.

Reply:
Go ahead. I will review in the morning.